doc/cookbook/source/examples/observers/least_angre_regression.rst

```diff
@@ -0,0 +1,50 @@
+=================================
+Observable Least Angle Regression
+=================================
+
+In this example, we will show how it is possible to obtain useful information from
+a Shogun's model while it is training. Here we will train a Least Angle
```

Comment:
don't mix tense: "we will train" or "we first create" (below). I would just stick to present tense